packages/blocks/src/embed-synced-doc-block/embed-synced-doc-block.ts

```diff
@@ -464,7 +464,8 @@ export class EmbedSyncedDocBlockComponent extends EmbedBlockElement<
     const isSelected = !!this.selected?.is('block');
     const theme = getThemeMode();
     const pageSpec = SpecProvider.getInstance().getSpec('page').value;
-    const edgelessSpec = SpecProvider.getInstance().getSpec('edgeless').value;
+    const edgelessSpec =
+      SpecProvider.getInstance().getSpec('edgeless:preview').value;
```

Comment:
If you need to add `edgeless:preview`, we'd better to change the current `preview` into `page:preview`